SCons/Tool/msvc.py

```diff
@@ -44,9 +44,11 @@
 import SCons.Warnings
 import SCons.Scanner.RC
 
-from .MSCommon import msvc_exists, msvc_setup_env_once, msvc_version_to_maj_min, msvc_find_vswhere
+from .MSCommon import msvc_setup_env_tool, msvc_setup_env_once, msvc_version_to_maj_min, msvc_find_vswhere
 from .MSCommon.common import get_pch_node
 
+tool_name = os.path.splitext(os.path.basename(__file__))[0]
```

Comment:
can't we just hardcode these? much simpler to read.

Reply:
Sure.  I was avoiding hard-coding the name since the tool name is derived from filename.